rest/admin_api.go

```diff
@@ -443,10 +445,25 @@ func (h *handler) handleGetStatus() error {
 			lastSeq, _ = database.LastSequence()
 		}
 
+		// Add replication status and configuration to the status response, per database.
+		replicationsStatus, err := database.SGReplicateMgr.GetReplicationStatusAll(h.getReplicationStatusOptions())
```

Comment:
We don't expect sgcollect to customise the query params, but using the default options makes sense here.

```suggestion
		replicationsStatus, err := database.SGReplicateMgr.GetReplicationStatusAll(
			db.DefaultReplicationStatusOptions(),
		)

```

Reply:
Makes sense, committed this change.